### ISSUE TITLE
swap keys with values in twigtemplates for a wider range of character in options

### ISSUE THREAD

**Issue by Puddingboy** [bolt/boltforms#147]
In the boltforms.bolt.yml a choices hash looks like this:
`{ submitted_value: option_text, other_values }`
This is the opposite of syntax in the override parameter in the twig template where: 
`{ option_text: submitted_value, other_values }`. This limits the amount of characters since hash keys are not allowed to have as many characters.

I want to propose to flip the twig template override choices parameter around to:
`{ submitted_value: option_text, other_values }`. 
This would allow developers to do this:
`{ false: "please choose", record_id: record_field, other_values }`
from the email/email.twig you can then do: 
`{ setcontent variable = "contenttype" }`.
in email/_blocks.twig you then can use something like the fieldname or fieldtype to see if a choice needs to have a converted value:
```
{% if 'contenttype_choice' in fieldname %}
    {{ contenttype_variable[value].recordfield }}
{% endif %}
```

tl;dr: flip the hashes in the override choices so that the yml choices and twig choices hash follow the same logic

**Comment by Puddingboy** [comment]
This request was wrong. There is nothing wrong with the current key/value composition. The current order aligns with the symfony spec. The reason i ran into issues was because my keys and values were poor syntax.

for reference:
`{ (record.title) : record.id }` now the () part gets parsed to become a keyword, but to do this properly, the syntax should look like this: `{ (record.title ~ '') : record.id ~ ' ' }` . this merges the values with an empty string composing string based keys and values.

**Comment by bobdenotter** [comment]
Thanks for the update! Very helpful if other people find this by Googling. 

I'll close this issue now, as no action is required, right? :-)
